pkg/proxy/ipvs/proxier.go

```diff
@@ -648,10 +648,19 @@ func (em proxyEndpointsMap) unmerge(other proxyEndpointsMap) {
 }
 
 // CanUseIPVSProxier returns true if we can use the ipvs Proxier.
-// This is determined by checking if all the required kernel modules are loaded. It may
+// This is determined by checking if all the required kernel modules can be loaded. It may
 // return an error if it fails to get the kernel modules information without error, in which
 // case it will also return false.
 func CanUseIPVSProxier() (bool, error) {
+	// Try to load IPVS required kernel modules using modprobe
+	for _, kmod := range ipvsModules {
+		err := utilexec.New().Command("modprobe", "--", kmod).Run()
```

Comment:
Could you please add some explanation on `--`?

Reply:
It is not required. For example, if you want to remove a file called `-f`, you cannot delete it with `rm -f` because `-f` will be treated as an option not a filename. You have to do it with `rm -- -f`.
The double dash is not something much here, just my personal habit.
https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean-also-known-as-bare-double-dash.